darion/api/handlers.go

```diff
@@ -39,8 +39,8 @@ func LogHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func logHandler(w http.ResponseWriter, r *http.Request, containerID, fileName string) {
-	filePath := filepath.Join(conf.ContainersHome, containerID, "logs", fileName)
-	fout, err := os.Open(filePath)
+	logPath := filepath.Join(buildLogLocationBase(containerID), fileName)
```

Comment:
I don't know if the cloned version is meant to be multi-tenant, or single-tenant. If it's meant to be single-tenant (i.e. Darion per container), then you should remove everything around containerID.

Reply:
@sargun the goal of this PR is to fudge the darion code just enough that allows both multi-tenant and single-tenant to work (initially), so I can work on injecting it into the container.  This is not meant to be the final code.  I don't want to spend time refactoring this code (just yet).  Rather, I think it's better to focus on the process injection which has more unknowns.

Reply:
Then I would keep this on your own branch for now, and wait to merge until it's ready, because this part of the code isn't touched very much.